### DISCUSSION TITLE
Changing a community name

### DISCUSSION THREAD

**Discussion by ghost** [Playabl-io/playabl#226] [General]
Hi! 
I made not one, but two typos while typing the name of my community. I cannot find a way to change it (nor to delete the community), any chance?

**Comment by jongrim** [comment]
Yeah, sorry @GKumquat, there's not currently a way to do this self-service. [I have a placeholder to add this functionality](https://github.com/Playabl-io/playabl/issues/132) and expect to knock it out in early 2024 as I implement some quality of life improvements.

In the meantime, I can manually update the name in the database if you'd like so that you don't have to wait on this. What is the current community name, and what would you like it updated to?

**Comment by ghost** [comment]
I mean, no need to be sorry, this is totally and uniquely my mistake :)

It would be great If you could change it. Now it's listed as "Utrecht RPG Exravaganza", I would like to change it to "Utrecht RPG Extravaganza!", that is, extravaganza spelled correctly and with a exclamation mark at the end.

Thanks for the help and sorry for the extra work!

**Reply by jongrim** [reply]
Done, and no trouble at all. Cool to see you're in Utrecht! Ik woon in Amsterdam! Do you play in Dutch? Ik leer Nederlands, maar ben ik een beginner. Misschien ik kun speel met jij op een dag. Cheers!

**Reply by ghost** [reply]
Leuk! We spelen in het Engels, maar ik wil ook proberen om in het Nederlands te spelen als een moeilijke taaloefening :)

Cheers, thanks for the quick reply. Keep an eye on our group for the next event then, hope to play with you one day.
